docs/userstories/ClusterNetworkPolicy/Zero_Trust_Cluster.md

```diff
@@ -0,0 +1,20 @@
+### Summary
+
+As a cluster admin, I want all Pods in my cluster to have a default zero-trust
+deny-all security posture, so that all traffic allowed in the cluster must be
+explicitly called out (by separate higher precedence policies).
```

Comment:
an example of such exceptions would provide better context, as was brought up by Sanjeev in the meeting (eg open traffic to prometheus etc)

Reply:
Done